### PR TITLE
Setup Route53 service

### PR DESCRIPTION
**Pull request by jderusse**
In order to help #999 this PR adds a new Route53 service (with a single operation `ListHostedZones`)

Once merge, It should ease integration of new operations.

### CODE REVIEW

**Comment by Nyholm** [comment]
Why did the tests fail on 7.2 only?

**Comment by jderusse** [comment]
> Why did the tests fail on 7.2 only?

Because of `prefer-lowest`

File: couscous.yml [hunk 1]
```diff
@@ -67,6 +67,9 @@ menu:
             rekognition:
                 text: Rekognition
                 url: /clients/rekognition.html
+            route53:
+                text: Lambda
```
Comment (misantron) [inline couscous.yml:71]:
```suggestion
                text: Route53
```
typo?

Comment (jderusse) [reply couscous.yml]: Good catch!!!!! thank you

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**